lib/Serverless.js

```diff
@@ -168,7 +168,7 @@ class Serverless {
     }
   }
 
-  run() {
+  async run() {
```

Comment:
It'll be nice to fully refactor this method to async/await (just with other commit or PR)

Reply:
totally, I didn't want to introduce it as a part of this PR, but another commit makes sense, I'll do it :+1:

Reply:
Yes, usually in such cases I think it's good to _precede_ addition of new functionality with commit or PR that refactors updated functionality to async/await.

Adding async/await syntax to not refactored parts looks confusing.

Reply:
Added that as a part of separate PR (which should be merged first): https://github.com/serverless/serverless/pull/8749